src/scenes/wizards_waltz.inc.c

```diff
@@ -1,37 +1,316 @@
-#include "asm/scenes/wizards_waltz/asm_080449a4.s"
+// For readability. !TODO - CHANGE/REMOVE
+#define GameInfo D_030055d0->gameInfo.wizardsWaltz

```

Comment:
I'm worried that a generic name like this might cause conflicts with other files and having it UpperCamelCase conflicts with naming standards for variables, maybe rename it to something more specific like "gWizardsWaltzInfo"?

Reply:
Done.